File: task/lib/debug.c
#include <stdio.h>
#include "command.h"

// |--------( Test Methods )--------|
void task1()
{
    String command = (String)malloc(500);
    String name ;
    getProcessName(&name);
    sendSignal(selectedSignal(7),&command, name);
}

void task2()
{
    String command = (String)malloc(500);
    String name ;
    getProcessName(&name);
    stopProcess(&command,name);
}

void test3()
{
    String command = (String)malloc(500);
    String name ;
    getProcessName(&name);
    startProcess(&command,name);
}


void test4()
{
    String name = (String)malloc(500);

    strcpy(name,selectedSignal(10));

    printf("\n name = %s", name);


    // printf("\nyour choice: %d",optionValidation(0,6));
}

// |--------( Debug Methods )--------|

void debug(int task)
{
    printf("\n[!] Task%d:\n",task);
    switch(task)
    {
        case 1:
            task1();
            break;
        case 2:
        
            task2();
            break;
        case 3: 
            test3();
            break;
        case 4:
            test4();
            break;
    }
}

File: project/lib/Debug.java
package lib;

public class Debug
{

    public void startDebug(int task)
    {
        System.out.println("[!] Task"+task+"\n");
        switch(task)
        {
            case 1:
                task1();
                break;
            case 2:
                task2();
                break;
            case 3: 
                task1();
                task2();
                break;
            
        }
    }


    // |----------( Task methods )----------|
    void task1()
    {
        System.out.println("Option menu");
    }

    void task2()
    {
        System.out.println("validation menu");
    }
}
